containers/singularity.def

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+BootStrap: docker
+From: continuumio/anaconda
+
+%runscript
+    export PATH=/opt/conda/bin:${PATH}
+    source activate unified
+    /bin/bash
+
+%post
+    apt-get update && apt-get -y install wget build-essential
+    export PATH=/opt/conda/bin:${PATH}
+
+    conda create -n unified -c conda-forge -c e3sm -c cdat mesalib livvkit mpas_analysis e3sm_diags processflow jupyter nco
```

Comment:
```suggestion
    conda create -n unified -c conda-forge -c e3sm -c cdat mesalib livvkit mpas-analysis e3sm_diags processflow jupyter nco
```
This will pick up the version from `conda-forge` instead of `e3sm`